.github/workflows/build_dependencies.yml

```diff
@@ -165,6 +165,7 @@ jobs:
     - name: Build Cache
       run: |
         pre=$([[ "${{ inputs.build-type }}" != "Debug" ]] && echo "-o sisl:prerelease=${{ inputs.prerelease }}" || echo "")
+        sudo rm -rf $ANDROID_HOME
```

Comment:
in the failure CI , Create and Test Package is not scheduled ,so `sudo rm -rf /usr/local/lib/android/` will not be executed in that CI, which might case a no-space left issue.

here , I add this to `Build Cache` phrase, which will be definitely scheduled in every CI pipeline

@sanebay